integration_test/migration_test.dart

```diff
@@ -296,8 +294,6 @@ void main() {
         _publicKey: [_address],
       },
     );
-    expect(await storage.readCurrentConnection(), _connection);
-    expect(storage.currentConnection, _connection);
```

Comment:
Why did you just remove migrating current connection?

Reply:
Reverted.